python/cdp/evm_client.py

```diff
@@ -572,3 +577,202 @@ async def wait_for_user_operation(
             timeout_seconds,
             interval_seconds,
         )
+
+    async def get_quote(
+        self,
+        from_token: str,
+        to_token: str,
+        amount: str | int,
+        network: str,
+    ) -> "SwapQuote":
+        """Get a quote for swapping tokens.
+
+        Args:
+            from_token (str): The contract address of the token to swap from.
+            to_token (str): The contract address of the token to swap to.
+            amount (str | int): The amount to swap (in smallest unit or as string).
+            network (str): The network to get the quote for.
+
+        Returns:
+            SwapQuote: The swap quote with estimated output amount and route.
+
+        """
+        from cdp.actions.evm.swap.types import SwapQuote
+        from cdp.openapi_client.models.evm_swaps_network import EvmSwapsNetwork
+        from cdp.openapi_client.models.get_quote_response import GetQuoteResponse
+
+        # Convert amount to string if it's an integer
+        amount_str = str(amount)
+
+        # Normalize addresses to lowercase
+        from_address = from_token.lower()
+        to_address = to_token.lower()
+
+        # Convert network to EvmSwapsNetwork enum
+        network_enum = EvmSwapsNetwork(network)
+
+        # Get quote from API - use the raw response to avoid oneOf deserialization issues
+        response = await self.api_clients.evm_swaps.get_evm_swap_quote_without_preload_content(
+            network=network_enum,
+            buy_token=to_address,
+            sell_token=from_address,
+            sell_amount=amount_str,
+            taker="0x0000000000000000000000000000000000010000",  # Valid placeholder address
+        )
+
+        # Read and parse the response manually
+        import json
```

Comment:
could we add these and the imports below within this function to the top to make a bit cleaner? Or maybe even a helper function to handle this logic so the evm_client is lighter?

Same comment for the other functions. It also seems like they have some common error handling logic that could be refactored